src/components/gameScreen/gameScreen.container.js

```diff
@@ -6,33 +6,37 @@ import {observer} from 'mobx-react-lite';
 import {AnswerItem} from './components/answerItem';
 import {GameScreenComponent} from './gameScreen.component';
 import AsyncStorageService from '../../utils/asyncStorage/asyncStorageService';
-import store from '../../store/store';
 import bookmarkStore from '../../store/bookmarkStore';
+import questionSetStore from '../../store/questionSetStore';
+import questionsStore from '../../store/questionsStore';
+import answersStore from '../../store/answersStore';
+import dialogsStore from '../../store/dialogsStore';
+import categoriesStore from '../../store/categoriesStore';
 
 export const GameScreenContainer = observer(() => {
   const navigation = useNavigation();
   const route = useRoute();
   const {questionNumber, headerBackground, score} = route.params;
-  const numberOfQuestions = store.chosenQuestionsSet.questions.length;
+  const numberOfQuestions = questionSetStore.chosenQuestionsSet.questions.length;
 
   const translateX = useSharedValue(400);
   const helpOpacity = useSharedValue(0);
 
   useEffect(() => {
     (async() => {
       try {
-        await store.setCurrentQuestion(store.chosenQuestionsSet.questions[questionNumber - 1]);
-        await store.setCurrentAnswers();
+        await questionsStore.setCurrentQuestion(questionSetStore.chosenQuestionsSet.questions[questionNumber - 1]);
+        await answersStore.setCurrentAnswers();
       } catch(e) {
         console.log('setCurrentQuestion or setCurrentAnswer error ', e);
       }
     })();
-    store.setIsAnswerHasChosen(false);
+    answersStore.setIsAnswerHasChosen(false);
     translateX.value = withSpring(0);
   }, []);
 
   const handleNextQuestion = (isNextButton) => {
-    if ((store.chosenMode === 'Hard') || isNextButton) {
+    if ((dialogsStore.chosenMode === 'Hard') || isNextButton) {
```

Comment:
same as above (but only create a variable inside of this function handleNextQuestion)